integration/test/mutation/main_test.go

```diff
@@ -0,0 +1,82 @@
+// +build k8srequired
+
+package mutation
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/giantswarm/apptest"
+	"github.com/giantswarm/micrologger"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/app-admission-controller/integration/env"
+)
+
+const (
+	prodCatalogName = "control-plane-catalog"
+	testCatalogName = "control-plane-test-catalog"
+)
+
+var (
+	appTest apptest.Interface
+	logger  micrologger.Logger
+)
+
+// TestMain allows us to have common setup and teardown steps that are run
+// once for all the tests https://golang.org/pkg/testing/#hdr-Main.
+func TestMain(m *testing.M) {
+	var err error
+
+	ctx := context.Background()
+
+	{
+		logger, err = micrologger.New(micrologger.Config{})
+		if err != nil {
+			panic(err.Error())
+		}
+	}
+
+	{
+		appTest, err = apptest.New(apptest.Config{
+			KubeConfigPath: env.KubeConfig(),
+			Logger:         logger,
+		})
+		if err != nil {
+			panic(err.Error())
+		}
+	}
+
+	{
+		values := `Installation:
+  V1:
+    Registry:
+      Domain: quay.io`
+		apps := []apptest.App{
+			{
+				CatalogName:   prodCatalogName,
+				Name:          "cert-manager-app",
```

Comment:
Sorry for my ignorance, but why we need `cert-manager-app` for app-admission-controller?

Reply:
The webhooks have to be served over TLS. So we use our cert-manager to issue a self signed cert.

Reply:
I added a comment for this. 👍

Reply:
Nice, Thanks!